dopamine/experiment5/experiment5_varplots.jl

```diff
@@ -0,0 +1,176 @@
+dopamineMT = filter(:worm => x -> x == "MT", dopamine)
+
+# GET MEAN AND STD OF MEAN SPEED AND SEM OF EACH BIN OF EACH CONDITION
+# x = mean of mean speed
+# y = mean of std speed
+# error = sem of std speed
+bufferN2binned = combine(groupby(bufferN2, [:bin]), :meanspeed => mean => :meanofmeanspeed, :stdspeed => mean => :meanofstdspeed, :stdspeed => sem => :semofstdspeed, :track => length => :n)
+bufferCBbinned = combine(groupby(bufferCB, [:bin]), :meanspeed => mean => :meanofmeanspeed, :stdspeed => mean => :meanofstdspeed, :stdspeed => sem => :semofstdspeed, :track => length => :n)
+bufferMTbinned = combine(groupby(bufferMT, [:bin]), :meanspeed => mean => :meanofmeanspeed, :stdspeed => mean => :meanofstdspeed, :stdspeed => sem => :semofstdspeed, :track => length => :n)
+dopamineN2binned = combine(groupby(dopamineN2, [:bin]), :meanspeed => mean => :meanofmeanspeed, :stdspeed => mean => :meanofstdspeed, :stdspeed => sem => :semofstdspeed, :track => length => :n)
+dopamineCBbinned = combine(groupby(dopamineCB, [:bin]), :meanspeed => mean => :meanofmeanspeed, :stdspeed => mean => :meanofstdspeed, :stdspeed => sem => :semofstdspeed, :track => length => :n)
+dopamineMTbinned = combine(groupby(dopamineMT, [:bin]), :meanspeed => mean => :meanofmeanspeed, :stdspeed => mean => :meanofstdspeed, :stdspeed => sem => :semofstdspeed, :track => length => :n)
+
+
+
+# PLOT
+
+# scatter of individual mean speeds
+fig1 = Figure(
+)
+
+ax1a = Axis(
+    fig1[1,1],
+    title = "Buffer",
+    xlabel = "Mean speed (µm/s)",
+    ylabel = "Std speed (µm/s)",
+)
+
+N2 = scatter!(ax1a, bufferN2.meanspeed, bufferN2.stdspeed, color = :pink)
+CB = scatter!(ax1a, bufferCB.meanspeed, bufferCB.stdspeed, color = :lightgreen)
+MT = scatter!(ax1a, bufferMT.meanspeed, bufferMT.stdspeed, color = :lightblue)
+
+ax1b = Axis(
+    fig1[1,2],
+    title = "Dopamine",
+    xlabel = "Mean speed (µm/s)",
+    ylabel = "Std speed (µm/s)",
+)
+
+scatter!(ax1b, dopamineN2.meanspeed, dopamineN2.stdspeed, color = :pink)
+scatter!(ax1b, dopamineCB.meanspeed, dopamineCB.stdspeed, color = :lightgreen)
+scatter!(ax1b, dopamineMT.meanspeed, dopamineMT.stdspeed, color = :lightblue)
+
+linkyaxes!(ax1a, ax1b)
+hideydecorations!(ax1b, grid = false)
+linkxaxes!(ax1a, ax1b)
+
+Legend(fig1[2, :],
+    [N2, CB, MT],
+    ["wild type", "cat-2 CB", "cat-2 MT"],
+    "Worm strain",
+    orientation = :horizontal,
+    titleposition = :left)
+
+# save(joinpath(experimentdir, "fig101.png"), fig1)
+
+
+
+# scatter and line of binned mean speeds
+fig2 = Figure(
+)
+
+ax2a = Axis(
+    fig2[1,1],
+    title = "Buffer",
+    xlabel = "Mean speed (µm/s)",
+    ylabel = "Std speed (µm/s)",
+)
+
+N2 = scatter!(ax2a, bufferN2binned.meanofmeanspeed, bufferN2binned.meanofstdspeed, color = :pink)
+CB = scatter!(ax2a, bufferCBbinned.meanofmeanspeed, bufferCBbinned.meanofstdspeed, color = :lightgreen)
+MT = scatter!(ax2a, bufferMTbinned.meanofmeanspeed, bufferMTbinned.meanofstdspeed, color = :lightblue)
```

Comment:
Looks like you don't use these assignments, then overwrite them below on lines 153:155

Reply:
These are for defining the [Legend](https://github.com/Klepac-Ceraj-Lab/Ce-DataAnalysis/pull/27/files#diff-bfd67b2191e6b5081daf4162d8554abc17824dd0e886ef824f6852c6a911d47dR161-R162). But yes I do overwrite them oops. I just deleted the second definition.